sapl/templates/materia/materialegislativa_filter.html

```diff
@@ -215,6 +243,14 @@ <h3>{% trans 'Pesquisa concluída com sucesso! Foi encontrada 1 matéria.'%}</h3
   {% block extra_js %}
 
   <script type="text/javascript" >
+  $( document ).ready(function() {
+    const links_votacao = document.querySelectorAll('.link_votacao_nominal');
+    links_votacao.forEach(link => {
+      link.addEventListener('click', function(event) {
+        event.preventDefault();
+      });
+    });
+  });
```

Comment:
Eu pensei ter feito esta recomendação no PR anterior, mas me enganei.

O jQuery pode simplificar esse trecho:

```suggestion
  $( document ).ready(function() {
      $('.link_votacao_nominal').on('click', function(event) {
        event.preventDefault();
      });
  });
```